pkg/cloudprovider/providers/aws/aws.go

```diff
@@ -43,10 +44,9 @@ import (
 	"github.com/aws/aws-sdk-go/service/elbv2"
 	"github.com/aws/aws-sdk-go/service/kms"
 	"github.com/aws/aws-sdk-go/service/sts"
-	"gopkg.in/gcfg.v1"
```

Comment:
Why have you removed this dependency?
This is still used [here](https://github.com/kubernetes/kubernetes/pull/69263/files#diff-07ba008af9c76b0539556ff7fde3105eR1114) and fails on `make quick-release`:
```
# k8s.io/kubernetes/pkg/cloudprovider/providers/aws
pkg/cloudprovider/providers/aws/aws.go:1114:9: undefined: gcfg
!!! [0328 11:47:12] Call tree:
!!! [0328 11:47:12]  1: /go/src/k8s.io/kubernetes/hack/lib/golang.sh:620 kube::golang::build_some_binaries(...)
!!! [0328 11:47:12]  2: /go/src/k8s.io/kubernetes/hack/lib/golang.sh:765 kube::golang::build_binaries_for_platform(...)
!!! [0328 11:47:12]  3: hack/make-rules/build.sh:27 kube::golang::build_binaries(...)
!!! [0328 11:47:12] Call tree:
!!! [0328 11:47:12]  1: hack/make-rules/build.sh:27 kube::golang::build_binaries(...)
!!! [0328 11:47:12] Call tree:
!!! [0328 11:47:12]  1: hack/make-rules/build.sh:27 kube::golang::build_binaries(...)
Makefile:92: recipe for target 'all' failed
make[1]: *** [all] Error 1
Makefile:487: recipe for target 'cross' failed
make: *** [cross] Error 1
!!! [0328 11:47:13] Call tree:
!!! [0328 11:47:13]  1: build/../build/common.sh:519 kube::build::run_build_command_ex(...)
!!! [0328 11:47:13]  2: build/release.sh:36 kube::build::run_build_command(...)
make: *** [Makefile:430: quick-release] Error 1
```

Reply:
This was VS Code being "helpful." It "cleaned" the imports when I committed. I'll restore and push now, sorry for that and good catch.